docs/standards/standard-detection.md

```diff
@@ -45,12 +45,12 @@ import UniversalProfile from '@lukso/lsp-smart-contracts/artifacts/UniversalProf
 import Web3 from 'web3';
 
 // Connect to the LUKSO L14 network
-const web3 = new Web3('https://rpc.l14.lukso.network');
+const web3 = new Web3('https://rpc.testnet.lukso.network');
 
 // Create an instance of the Universal Profile
 const myUPContract = new web3.eth.Contract(UniversalProfile.abi, '<contract-address>');
 
-const ERC725AccountInterfaceId = '0x63cb749b';
+const ERC725AccountInterfaceId = '0x3e89ad98';
```

Comment:
maybe we should rename this constant `lsp0InterfaceId` or `lsp0Erc725AccountInterfaceId`? To make it match more with the table of interface IDs.